scripts/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Configure to exit script as soon as a command fails.
+set -o errexit
+
+# Clean the existing build directory.
+rm -rf build
+
+# Create a temporary directory to place ignored files (e.g. examples).
+tmp_dir="$(mktemp -dp.)"
+
+# Move the ignored files to the temporary directory.
+while IFS="" read -r ignored
+do
+  mv "contracts/$ignored" "$tmp_dir"
+done < contracts/.npmignore
```

Comment:
I thought this was the most future-proof way of doing this instead of hard-coding the `mocks` and `examples` directories. It's still not 100% accurate, though, because there could technically be other ignored things in different `.gitignore` or `.npmignore` files around the repo, but I think we'll be fine.